### PR TITLE
Added descriptive text about configuration/actuation usage

### PR DESCRIPTION
**Pull request by akeranen**
Added descriptive text about configuration/actuation usage

### CODE REVIEW

**Comment by akeranen** [comment]
Addressing issue #61

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -416,6 +416,15 @@ to describe that a resource is available in a SenML format in the
 first place. The relevant media type indicator is included in the
 Content-Type (ct=) attribute.
 
+## Configuration and Actuation usage
+
+While typical use of SenML is retrieving sensor data, SenML can also
```
Comment (fluffy) [inline draft-ietf-core-senml.md]: I'd like to delete the "While typical use of SenML is retrieving sensor data," and juse leave and not get into what is typical.

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -416,6 +416,15 @@ to describe that a resource is available in a SenML format in the
...
+be used for configuring parameters and controlling actuators. When a
+SenML Pack is sent (e.g., using a HTTP/CoAP POST or PUT method) and
+the semantics of the target are such that SenML is interpreted as
+configuration/actuation, SenML Records are interpreted as a request to
+change the values of given resources (names) to given values at given
```
Comment (fluffy) [inline draft-ietf-core-senml.md]: perhaps change "at given" -> "at the given"

**Review by fluffy — APPROVED** [review]
This looks good to me. Two little tweeks but if you don't like them, I can live with it as it is.

File: draft-ietf-core-senml.md [hunk 3]
```diff
@@ -416,6 +416,14 @@ to describe that a resource is available in a SenML format in the
 first place. The relevant media type indicator is included in the
 Content-Type (ct=) attribute.
 
+## Configuration and Actuation usage
+
+SenML can also be used for configuring parameters and controlling
+actuators. When a SenML Pack is sent (e.g., using a HTTP/CoAP POST or
+PUT method) and the semantics of the target are such that SenML is
+interpreted as configuration/actuation, SenML Records are interpreted
+as a request to change the values of given resources (names) to given
```
Comment (cabo) [inline draft-ietf-core-senml.md:425]: We could say "resources or subresources (as identified by their names)" (the names do not always identify actual separate resources).

**Pull request merged**